pcdsdevices/signal.py

```diff
@@ -352,7 +352,7 @@ class UnitConversionDerivedSignal(DerivedSignal):
     def __init__(self, derived_from, *,
                  derived_units: str,
                  original_units: typing.Optional[str] = None,
-                 user_offset: typing.Optional[typing.Any] = None,
+                 user_offset: typing.Optional[typing.Any] = 0,
```

Comment:
Nitpicky, but can this really be `typing.Any`?

Reply:
`{float, int, np.int*, np.float*}` -> `numbers.Real`, I suppose is most accurate